### ISSUE TITLE
Magic Command Does Nothing

### ISSUE THREAD

**Issue by thejohnhoffer** [jupyter/jupyter_console#121]
Hi. The `%load` command does nothing for `v5.1.0` on `MacOS`

```
hoffbuch:2017 John$ jupyter-console
Jupyter console 5.1.0

Python 3.6.0 (default, Jan 13 2017, 20:42:53)
Type "copyright", "credits" or "license" for more information.

IPython 5.3.0 -- An enhanced Interactive Python.
...

In [1]: import cv2

In [2]: %load 1

In [3]:      
```

**Comment by thejohnhoffer** [comment]
For comparison:

```
hoffbuch:2017 John$ ipython
Python 3.6.0 (default, Jan 13 2017, 20:42:53)
Type "copyright", "credits" or "license" for more information.

IPython 5.3.0 -- An enhanced Interactive Python.
...

In [1]: import cv2

In [2]: %load 1

In [3]: # %load 1
   ...: import cv2
```

**Comment by takluyver** [comment]
Can you see if #123 fixes it?

**Comment by takluyver** [comment]
I'm assuming that this is fixed by #123. That will be in jupyter_console 5.2. Let me know if the issue still occurs with a version newer than that (or in master).
